### PR TITLE
[Extended team WIP] ALMS-203 [iOS] Update quickstart app to use latest versions of Auth and Tracking SDKs

### PR DESCRIPTION
**Pull request by zeeshanmakeen**

### CODE REVIEW

**Review by cgalvan — APPROVED** [review]
Nitpick: I would change this to using WIP new version vs. latest version of SDK, since they are pointing to branches on a fork, instead of the actual repo for those SDKs.

**Comment by mbalfour-amzn** [comment]
Going forward, please put "Extended team WIP" as a label instead of in the title. Thanks!

File: tracking-with-geofence-notifications/AWSLocationSampleApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      }
+    },
+    {
+      "identity" : "amazon-location-mobile-tracking-sdk-ios",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/makeen-project/amazon-location-mobile-tracking-sdk-ios.git",
+      "state" : {
+        "branch" : "iot_mqtt_feature",
+        "revision" : "f42ca3f05eb609b3bcc27497e57a1d368d4f430a"
+      }
```
Comment (mbalfour-amzn) [inline tracking-with-geofence-notifications/AWSLocationSampleApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:20]: These should be coming from aws-geospatial, not makeen-project

**Review by mbalfour-amzn — CHANGES_REQUESTED** [review]

**Comment by cgalvan** [comment]
Closing this in favor of https://github.com/aws-geospatial/amazon-location-samples-ios/pull/4

**Pull request closed without merging**